src/raftstore/store/engine.rs

```diff
@@ -73,6 +73,20 @@ pub trait Peekable {
         Ok(Some(m))
     }
 
+    fn get_msg_cf<M>(&self, cf: &str, key: &[u8]) -> Result<Option<M>>
```

Comment:
please add test for all new added functions. 

Btw, I think we can use this function to simplify get_msg.